### ISSUE TITLE
evalutating only the first n classifiers in an ensemble

### ISSUE THREAD

**Issue by andreh7** [dmlc/xgboost#12]
is it possible to use only the first n classifiers (e.g. trees) in the ensemble ? Is it possible to do this from python ?

(this would be interesting in order to see at which point the 'loss' on the test set increases again or becomes larger than the loss on the training set)
